packages/main/src/plugin/extension-loader.ts

```diff
@@ -212,6 +213,23 @@ export class ExtensionLoader {
     this.moduleLoader.addOverride({ '@podman-desktop/api': ext => ext.api }); // add podman desktop API
 
     this.moduleLoader.overrideRequire();
+    // register configuration for the max activation time
+    const maxActivationTimeConfiguration: IConfigurationNode = {
+      id: 'preferences.extension',
+      title: 'Extensions',
+      type: 'object',
+      properties: {
+        [ExtensionLoaderSettings.SectionName + '.' + ExtensionLoaderSettings.MaxActivationTime]: {
+          description: 'Maximum activation time for an extension, in seconds.',
+          type: 'number',
+          default: 5,
```

Comment:
Default is still 5, should we do 10 like discussed on the call?

Reply:
changed to 10